js/app.js

```diff
@@ -4,14 +4,14 @@ var prevNum = [];
 var currNum = [];
 var objStore = [];
 var isSame = false;
```

Comment:
You may want to use local variables instead of manipulating a global variable like this. That way each function is more self contained and does not depend on the current value of `isSame` in order to work properly.